cypress/integration/dateUtils.spec.js

```diff
@@ -0,0 +1,43 @@
+const { addYearToDate } = require('../../server/utils');
```

Comment:
This is related to cypress / e2e tests, not the right place for unit tests. Remove this from here.